src/world/Server/Packets/Handlers/GuildHandler.cpp

```diff
@@ -424,7 +424,8 @@ void WorldSession::HandleGuildRank(WorldPacket& recv_data)
                               pRank->iTabPermissions[4].iFlags, pRank->iTabPermissions[4].iStacksPerDay,
                               pRank->iTabPermissions[5].iFlags, pRank->iTabPermissions[5].iStacksPerDay);
 
-    _player->m_playerInfo->guild->SendGuildRoster(this);
+    _player->GetGuild()->SendGuildQuery(NULL);
```

Comment:
Use nullptr instead of NULL....
NULL is defined as an integer (macro) an is guaranteed to initialize pointers to a null pointer (much like nullptr).
nullptr is prvaalue of specific types, so it should be prefered.

Long story short: Use nullptr instead of NULL (the function requires a pointer :SendGuildQuery(WorldSession* pClient)  )

Check out C++11 features for further information.